modules/katello_content_view.py

```diff
@@ -97,9 +98,7 @@ def find_organization(self, name):
             self._module.fail_json(msg="No organization found for %s" % name)
 
     def find_product(self, name, organization):
-        org = self.find_organization(organization)
```

Comment:
We should only resolve it once (one API call). Then use the entity object after that in memory.